### DISCUSSION TITLE
configure modoboa to use a mail relay to send mail

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [modoboa/modoboa#3179] [Help]
HI, i'm struggling to find out how to configure modoboa to use an external relay
the documentation say there's a plugin but i can't seem to find where to access the extensions from the admin webgui?
or am i supposed to just configure postfix main.cf with the info?

**Comment by tonioo** [comment]
There is no plugin for that yet. You need to update postfix configuration directly
